### PR TITLE
feat: Business Card Component 

### PR DESCRIPTION
**Pull request by Ktoxcon**

### CODE REVIEW

File: app/src/ui/business-card/BusinessCard.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  const redirectToBusinessPage = (_event: React.MouseEvent<HTMLElement>) => {
+    router.push(pageLink);
+  };
+
+  return (
+    <div className={clsx(styles["business-card"], className)} onClick={redirectToBusinessPage}>
+      <Container>
+        <Grid.Row>
+          <Grid.Col sm={6} lg={4} xl={6}>
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:35]: @Ktoxcon yes, this is the way. The Card will take the column space, and we don't mess with static `width` s

File: app/src/ui/business-card/BusinessCard.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <Card shadow>
+              <Card.Content>
+                <img
+                  className={styles["business-card__cover"]}
+                  src={content?.cover}
+                  alt={`A representative image of ${content?.name}`}
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:41]: > Should alternative imagine text be included in the internationalization process?

In an ideal context yes. But let's ask a Junior dev to do it in the future 😅

File: app/src/ui/business-card/BusinessCard.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                />
+                <Typography.Headline4>{content?.name}</Typography.Headline4>
+                <Typography.Subtitle>{content?.location}</Typography.Subtitle>
+                <Typography.Description>{content?.description}</Typography.Description>
+                <Container fluid>
+                  <Grid.Row>
+                    <Grid.Col className={styles["business-card__short"]}>
+                      <Typography.Headline6>{truncateRaisedAmount(content.raised)}</Typography.Headline6>
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:49]: "amount raised" will come from `business_funding_campaign_plan.total_sats_invested` 👍 

You will fetch this data from the parent container, like this: https://github.com/bancosatoshi/web/blob/sequelize/app/src/app/business/BusinessDetails/BusinessDetailsContainer.tsx#L19

File: app/src/ui/business-card/BusinessCard.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                      <Typography.MiniDescription>Raised</Typography.MiniDescription>
+                    </Grid.Col>
+                    <Grid.Col className={styles["business-card__short"]}>
+                      <Typography.Headline6>{content?.investors}</Typography.Headline6>
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:53]: Same, comes from database table: `business_funding_campaign_transactions`
You should filter by `business_id` and remove `user_id` duplicates, and get the count.

File: app/src/ui/business-card/BusinessCard.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                      <Typography.MiniDescription>
+                        {content?.investors > 1 ? "Investors" : "Investor"}
+                      </Typography.MiniDescription>
+                    </Grid.Col>
+                  </Grid.Row>
+                  <Visible lg xl>
+                    <Grid.Row>
+                      <Grid.Col className={styles["business-card__short"]}>
+                        <Typography.Headline6>{content?.payback}</Typography.Headline6>
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:62]: This is data from `business_funding_campaign_plan.investment_multiple`

File: app/src/ui/business-card/BusinessCard.tsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                        <Typography.MiniDescription>Payback</Typography.MiniDescription>
+                      </Grid.Col>
+                      <Grid.Col className={styles["business-card__short"]}>
+                        <Typography.Headline6>{content?.daysLeft}</Typography.Headline6>
```
Comment (netpoe) [inline app/src/ui/business-card/BusinessCard.tsx:66]: This comes from a calculation of days left with `moment`, readind the date from `business_funding_campaign_plan .expires_at`

**Review by netpoe — CHANGES_REQUESTED** [review]
@Ktoxcon 
Let's merge #1 so you can build a query for this component data. 

Look into: https://docs.google.com/spreadsheets/d/1SOlJ81IR_pxGhWsoM7L61am54SqMO6KroIPLUpqFfQM/edit?usp=sharing for more details of the data structure.

File: app/src/ui/business-card/BusinessCard.tsx (hunk 4 above)
Comment (netpoe) [reply app/src/ui/business-card/BusinessCard.tsx:53]: https://github.com/bancosatoshi/web/blob/sequelize/app/src/pages/api/business/resolvers/queries/getBusinessCampaignBySlug.ts#L10

See how to do queries from a `graphql` resolver.

File: app/src/ui/business-card/BusinessCard.tsx (hunk 6 above)
Comment (Ktoxcon) [reply app/src/ui/business-card/BusinessCard.tsx:66]: @netpoe What about a Sequelize virtual for this one?

[Virtuals](https://sequelize.org/master/manual/getters-setters-virtuals.html)

Comment (netpoe) [reply app/src/ui/business-card/BusinessCard.tsx:66]: Wow! I wasn't aware of this. What are you thinking, `daysLeft`?

**Pull request closed without merging**